### PR TITLE
5158 actions and proxy

### PR DESCRIPTION
**Pull request by mrinsin**
Added actions and proxy for getting todays opinions and setting it in state

### CODE REVIEW

File: shared/src/proxies/public/getTodaysOpinionsProxy.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const { get } = require('../requests');
+
+// eslint-disable-next-line spellcheck/spell-checker
```
Comment (matthopson) [inline shared/src/proxies/public/getTodaysOpinionsProxy.js:3]: What was it mad about?

Comment (mrinsin) [reply shared/src/proxies/public/getTodaysOpinionsProxy.js:3]: it wanted me to write `today's` and not `todays` -_-

Comment (matthopson) [reply shared/src/proxies/public/getTodaysOpinionsProxy.js:3]: Based on the other descriptions I'm assuming "todaysOpinions"? We can add this word to to the spellchecker's whitelist

Comment (mrinsin) [reply shared/src/proxies/public/getTodaysOpinionsProxy.js:3]: ooh, I just learned something new! Thanks - I will do that

Comment (matthopson) [reply shared/src/proxies/public/getTodaysOpinionsProxy.js:3]: https://github.com/flexion/ef-cms/blob/develop/.eslintrc.js#L181

**Review by matthopson — APPROVED** [review]
Just the nit re: spellcheck, otherwise, looks good!

**Review by codyseibert — APPROVED** [review]

**Pull request merged**